tests/runner.rs

```diff
@@ -38,19 +38,29 @@ fn run_one(path: &path::Path) -> Result<(), io::Error> {
     first_line.pop();
     let path = path.into();
     let test_func = match first_line.as_str() {
+        // make sure the test compiles, but don't run it
         "// compile" => utils::assert_compiles,
+        // the test compiles, and don't require a `main` function
         "// no-main" => utils::assert_compiles_no_main,
+        // the test shouldn't compile
         "// fail" | "// compile-fail" | "// compile-error" => utils::assert_compile_error,
+        // it should compile, run, and exit successfully
         "// succeeds" => utils::assert_succeeds,
+        // it should compile successfully then crash at runtime
         "// crash" => utils::assert_crash,
+        // tests can only be ignored if they have an issue open on github
         "// ignore" => panic!("ignored tests should have an associated issue"),
         line => {
+            // `code: x` - it should compile, run, and exit with code 1
```

Comment:
This should probably say `[..] and exit with code x`, right?

Reply:
Yes it should!